src/app/auth/auth.module.ts

```diff
@@ -0,0 +1,30 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+
+import { AuthRoutingModule } from './auth-routing.module';
+import { AuthComponent } from './auth.component';
+import { LoginComponent } from './login/login.component';
+import { RegisterComponent } from './register/register.component';
+import { ReactiveFormsModule } from '@angular/forms';
+
+import {MatInputModule} from '@angular/material/input';
```

Comment:
should have space around to be
`import { MatInputModule } from '@angular/material/input';`